jetty-client/src/test/java/org/eclipse/jetty/client/ConnectionPoolTest.java

```diff
@@ -71,6 +71,12 @@
     {
         return Stream.of(
             new ConnectionPoolFactory("duplex", destination -> new DuplexConnectionPool(destination, destination.getHttpClient().getMaxConnectionsPerDestination(), destination)),
+            new ConnectionPoolFactory("duplex-maxDuration", destination ->
+            {
+                DuplexConnectionPool pool = new DuplexConnectionPool(destination, destination.getHttpClient().getMaxConnectionsPerDestination(), destination);
+                pool.setMaxDuration(10);
+                return pool;
+            }),
```

Comment:
I don't see a test that verifies that the max duration mechanism works?

Reply:
You're right, adding a specific test for this functionality would be good. Let me add one.